src/components/TaskHeader.js

```diff
@@ -60,7 +62,7 @@ function TaskHeader(props) {
                     >
                         <View style={[styles.flexRow, styles.alignItemsCenter, styles.justifyContentBetween, styles.pv3]}>
                             <View style={[styles.flexRow, styles.alignItemsCenter, styles.justifyContentBetween]}>
-                                {!_.isEmpty(props.report.managerID) && (
```

Comment:
I have no idea why this doesn't work...https://expensify.slack.com/archives/C03TQ48KC/p1686838071775199

Reply:
Hmm because it's a number, this works: `console.log(!_.isEmpty(props.report.managerID.toString()), props.report.managerID);`